charts/redpanda/values.schema.json

```diff
@@ -697,7 +687,6 @@
           "type": "object",
           "required": [
             "port",
-            "external",
```

Comment:
external is required. If it's missing `$listenerVals.external` will panic.

Reply:
I may have mistakenly removed `external` from here (rather than another place I meant to). I'll take a closer look and test it out again.